usecase/usecase-webapi-xwalk-tests/samples/GetUserMedia/LICENSE.md

```diff
@@ -0,0 +1,29 @@
+Copyright (c) 2014, The WebRTC project authors. All rights reserved.
```

Comment:
This license file is unnecessary as there is already a license note in each file you integrate. By the way, where is this sample integrated from?